util/crc32c_ppc.c

```diff
@@ -0,0 +1,101 @@
+/* Copyright (C) 2017 International Business Machines Corp.
+ * All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
```

Comment:
I just spoke to @antonblanchard who indicated you can merge this under any open source license you choose.